src/Core/Style.js

```diff
@@ -42,6 +42,18 @@ function getImageFromSprite(sprites, key) {
     return image;
 }
 
+const textAnchorPosition = {
```

Comment:
could you use anchor string to define properties and remove the switch case.
```js
    //...
    'bottom-left': [0, -1],
    'bottom-right': [-1, -1],
    //...
```

Reply:
Yes indeed, changing that